airflow-core/src/airflow/ui/public/i18n/locales/en/browse.json

```diff
@@ -19,6 +19,8 @@
     "columns": {
       "dag": "Dag",
       "key": "Key",
+      "task_display_name": "Task",
+      "timestamp": "Timestamp",
```

Comment:
`common:task_one`
and
`dashboard:timestamp` already exist so we can use those.